### ISSUE TITLE
Hope the pipeline can use the variable cards in the same network variable card box directl.y

### ISSUE THREAD

**Issue by Gan-N** [CyclopsMC/IntegratedTunnels#185]
#### Issue type:

- :heavy_plus_sign: Feature request

____

#### Short description:

When I add a variable card to the pipeline, it would be nice if I could choose the variable card that I put in my backpack or directly apply the variable card in the same network variable card box.

**Comment by rubensworks** [comment]
I don't understand what you mean, can you clarify?

**Comment by Gan-N** [comment]
When placing variable cards on the input and output pipes, I want to be able to directly reference the variable cards in the variable store under the same network.

**Comment by Gan-N** [comment]
When I add a variable card to the pipeline, it would be nice if I could choose the variable card that I put in my backpack or directly apply the variable card in the same network variable card box.

**Comment by rubensworks** [comment]
Sorry, I still don't understand.

**Comment by Gan-N** [comment]
I created a variable card with variable ID 20 and put it in the variable store

1. When I place an item exporter, I need to place a variable card for the item exporter. I can use the variable card with ID 20.

2. When I place the second item exporter, I can continue to use the variable card with ID 20



Sorry, I found that this idea is the same as proxy in integrated dynamics, but there is no need to make a proxy variable card
